static/app/stores/hookStore.tsx

```diff
@@ -49,6 +50,9 @@ const storeConfig: Reflux.StoreDefinition & HookStoreInterface = {
  *
  * This functionality is primarily used by the SASS sentry.io product.
  */
-const HookStore = Reflux.createStore(storeConfig) as Reflux.Store & HookStoreInterface;
+
+const HookStore = makeSafeRefluxStore(
+  Reflux.createStore(storeConfig)
+) as HookStoreInterface & SafeRefluxStore;
```

Comment:
This looks different than the others, shouldn't this be:
```suggestion
const HookStore = makeSafeRefluxStore(
  Reflux.createStore(storeConfig) as HookStoreInterface & SafeRefluxStore
);
```

?